WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -2007,6 +2008,7 @@ private enum Localization {
         "Cannot duplicate product",
         comment: "The title of the alert when there is an error duplicating the product"
     )
+    static let defaultTitle = NSLocalizedString("Product", comment: "Product title")
```

Comment:
nit: in a platform announcement in Oct 2023 p91TBi-aJl-p2, we're going to include a unique key in each new localizable string now.

```suggestion
    static let defaultTitle = NSLocalizedString("productForm.defaultTitle", value: "Product", comment: "Product title")
```

Reply:
Tnx!